tests/assistant_spec.js

```diff
@@ -193,6 +193,30 @@ describe('TranslationAssistant', function() {
             assertSuggestions(allItems, itemsToTranslate, translatedStrs);
         });
 
+        it('should return null when there\'s nl text inside \\text{}',
```

Comment:
These two tests actually pass even before the fix, not quite sure how, but I thought they should logically be here anyway.